File: 프로그래머스/0/120911. 문자열 정렬하기 （2）/문자열 정렬하기 （2）.cs
using System;

public class Solution 
{
    public string solution(string my_string) 
    {
        string str = my_string.ToLower();
        char[] str_list = str.ToCharArray();
        Array.Sort(str_list);
        string result = new string(str_list);
        return result;
    }
}

File: 프로그래머스/0/120911. 문자열 정렬하기 （2）/문자열 정렬하기 （2）.cpp
#include <string>
#include <vector>
#include <algorithm>


using namespace std;

string solution(string my_string) 
{
    for(char &c : my_string)
    {
        c = tolower(c);
    }
    sort(my_string.begin(), my_string.end());
    return my_string;
}